server/venueless/settings.py

```diff
@@ -136,22 +135,24 @@
     ALLOWED_HOSTS = [urlparse(SITE_URL).netloc]
 
 if os.getenv("VENUELESS_COOKIE_DOMAIN", ""):
-    SESSION_COOKIE_DOMAIN = os.getenv("VENUELESS_COOKIE_DOMAIN", "")
+    # SESSION_COOKIE_DOMAIN = os.getenv("VENUELESS_COOKIE_DOMAIN", "")
     CSRF_COOKIE_DOMAIN = os.getenv("VENUELESS_COOKIE_DOMAIN", "")
 
-SESSION_COOKIE_SECURE = (
-    os.getenv("VENUELESS_HTTPS", "True" if SITE_URL.startswith("https:") else "False")
-    == "True"
-)
+# SESSION_COOKIE_SECURE = (
+#     os.getenv("VENUELESS_HTTPS", "True" if SITE_URL.startswith("https:") else "False")
+#     == "True"
+# )
```

Comment:
You can probably just delete all the cookie settings right away, I don't think we'll need them any time soon?

Reply:
Probably, yes